Admin/MonitorExchangeAuthCertificate/MonitorExchangeAuthCertificate.ps1

```diff
@@ -434,6 +553,23 @@ function Main {
             Write-Host ""
             Write-Host ($multipleExchangeADSitesWording) -ForegroundColor Yellow
         }
+
+        if ((-not($WhatIfPreference)) -and
```

Comment:
We should use ShouldProcess, not manually check $WhatIfPreference. See the examples here: https://learn.microsoft.com/en-us/powershell/scripting/learn/deep-dives/everything-about-shouldprocess

Reply:
`$WhatIfPreference` is checked at several points in the script. Will open a work item to track the work to handle this better as suggested.

Reply:
https://github.com/microsoft/CSS-Exchange/issues/1487